CMakeLists.txt

```diff
@@ -90,12 +90,13 @@ add_definitions(-DVERSION="${VERSION}")
 set(CMAKE_AUTOMOC ON)
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 set(QT_MIN_VER 5.5.1) # The version in Ubuntu 16.04
-find_package(Qt5Core ${QT_MIN_VER} REQUIRED)
-find_package(Qt5Xml ${QT_MIN_VER} REQUIRED)
-find_package(Qt5Widgets ${QT_MIN_VER} REQUIRED)
-find_package(Qt5LinguistTools ${QT_MIN_VER} REQUIRED)
-find_package(Qt5Svg ${QT_MIN_VER} REQUIRED)
-find_package(Qt5Test ${QT_MIN_VER} REQUIRED)
+
+find_package(Qt6Core ${QT_MIN_VER} REQUIRED)
+find_package(Qt6Xml ${QT_MIN_VER} REQUIRED)
+find_package(Qt6Widgets ${QT_MIN_VER} REQUIRED)
+find_package(Qt6LinguistTools ${QT_MIN_VER} REQUIRED)
+find_package(Qt6Svg ${QT_MIN_VER} REQUIRED)
+find_package(Qt6Test ${QT_MIN_VER} REQUIRED)
```

Comment:
What happens on Qt5 now?

Reply:
I mean, we need to support both Qt5 and Qt6. I haven't tested any Qt6 stuff yet myself :)

Reply:
Ok. But it will be easier if you will use qt 5.15 as minimum required :) There are a lot of features with linked targets and functions name without major version

Reply:
I made a fix. But i didnt test it with Qt version less 5.15

Reply:
Good to know! However, the problem is that we need to support Ubuntu 18.04 and Ubuntu 20.04 (Ubuntu 16.04 will be EOL'ed this year) and they come with much older Qt 5. At this phase it would be the best to make Qt 6 completely optional and probably add some `#ifdef`'s to the code if needed.